File: Python/Priyanshi/DAY 5 Q2.py
num1= int(input ("Enter:"))  
num2 = int(input ("Enter:"))
print ("The Prime Numbers in the range are: ")  
for j in range (num1, num2 + 1):  
        for i in range (2, j):  
            if (j % i) == 0:  
                break
        else:
            print (j)

File: C&C++/MEHUL/Day5_Q2.c
#include <stdio.h>

int main()
{
    int num1, num2, i, lcm;

    printf("Enter two numbers: ");
    scanf("%d %d", &num1, &num2);

    if (num1 < num2)
        i = num1;
    else
        i = num2;

    while (i>=0)
    {
        if (i % num1 == 0 && i % num2 == 0)
        {
            lcm = i;
            break;
        }
        i++;
    }

    printf("The LCM of %d and %d is %d.\n", num1, num2, lcm);

    return 0;
}
